### DISCUSSION TITLE
Welcome to segment-anything-video Discussions!

### DISCUSSION THREAD

**Discussion by kadirnar** [kadirnar/segment-anything-video#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ANANAYAGG** [comment]
hello sir i want to know that till now you have created  the output masks of the image but in order to manipulate the masks and remove or move the layers how can we import it using sam api

**Reply by kadirnar** [reply]
I don't understand Do you want mask results(score,bbox,polygon)?Can you show example from original repo?
